service.subtitles.subdivx/addon.xml

```diff
@@ -21,4 +22,8 @@
     <email></email>
     <source>https://github.com/ramiro/service.subtitles.subdivx</source>
   </extension>
+  <news lang="es">v0.3.0
+  - Se agrega compatibilidad con Kodi 18 Leia. Necesita instalar manualmente el add-on "Archive support" (por spiff)</news>
+  <news lang="en">v0.3.0
+  - Added compatibility with Kodi 18 Leia. You need to install manually the "Archive support" (by spiff) add-on</news>
```

Comment:
`news` tag has to go inside `<extension point="(kodi|xbmc).addon.metadata">` and can't be translated